waku/node/v2/wakunode2.nim

```diff
@@ -189,6 +192,9 @@ proc publish*(node: WakuNode, topic: Topic, message: WakuMessage) =
   ## Status: Implemented.
   ##
 
+  # @TODO MAYBE THIS?
```

Comment:
Notify should happen where/when we receive the WakuFilter messages or in the WakuRelay handler I believe

Reply:
Oh, never mind. Misunderstood this. It is about sending the messages to the interested peers, not notifying on receival of messages.